lib/main_ltp.pm

```diff
@@ -113,91 +133,96 @@ sub loadtest_from_runtest_file {
     shutdown_ltp(run_args => testinfo($test_result_export));
 }
 
-# Replace loadtest_from_runtest_file with this to stress test reverting to
+# Replace loadtest_kernel_from_runtest_file with this to stress test reverting to
 # snapshots
 sub stress_snapshots {
     my $count = 100;
 
     for (my $i = 0; $i < $count / 2; $i++) {
         # This will always fail and revert to the previous milestone, which
         # will either be boot_ltp or write_random#$i
-        loadtest('run_ltp');
-        loadtest('write_random');
+        loadtest_kernel('run_ltp');
+        loadtest_kernel('write_random');
     }
 }
 
 sub load_kernel_tests {
-    load_bootloader_s390x();
-    loadtest "../installation/bootloader" if is_pvm;
+    if (get_var('LTP_BAREMETAL') && get_var('INSTALL_LTP')) {
+        load_baremetal_tests();
+    } else {
+        load_bootloader_s390x();
+    }
+
+    loadtest_kernel "../installation/bootloader" if is_pvm;
```

Comment:
Use regular `loadtest()` here?

Reply:
Well, yep I can remove custom function and add `"tests/kernel/"` for regular `loadtest()`. I don't have strong preference, both ways looks ok to me.

Reply:
No, I didn't want you to remove `loadtest_kernel()`. This specific line should just be `loadtest "installation/bootloader" if is_pvm;` (using the `loadtest()` from main_common.pm).

Reply:
But never mind, since there's no reason to refactor `loadtest()` in this PR now.

Reply:
Right :)